File: programmers/난이도별/level01.평균_구하기/JiaLee0707.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

double solution(int arr[], size_t arr_len) {
    double answer = 0;
    for(int i=0; i<arr_len; i++) {
        answer += arr[i];
    }
    return answer/arr_len;
}

File: programmers/난이도별/level01.최대공약수와_최소공배수/JiaLee0707.js
function solution(n, m) {
    let answer = [];
    let min = n > m ? m : n;
    let max = n > m ? n : m;
    
    // 1
    // for (let i=1; i<=min; i++) {
    //     if (n%i === 0 && m%i === 0) {
    //         answer[0] = i;
    //         answer[1] = i * (n/i) * (m/i)
    //     }
    // }
    
    // 2
    while (min != 0) {
        answer[0] = max % min;
        max = min;
        min = answer[0];
    }
    answer[0] = max;
    
    answer[1] = n*m/answer[0];
    
    return answer;
}
